### ISSUE TITLE
Priority mode is ignored

### ISSUE THREAD

**Issue by mantono** [WhisperSystems/Signal-Android#5436]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
This relates to issue #2548 - but I have the opposite problem. Even tough I have priority mode enabled I always get notifications (sound, vibration and notification icon) when receiving an SMS. I do not know how it is with regular Signal data messages as I don't know anyone else with Signal.

### Steps to reproduce
- In Signal go to _Settings_ --> _Notifications_ --> _Notifications_ --> [Enable]
- In Android select Priority mode.
- Android Settings --> _Sound & notification_ --> _Interruptions_ --> _Calls/messages from_ --> 
[Starred contacts only]
- Receive an SMS (from someone not marked as a favourite/starred contact).

**Actual result:** A notification is triggered, in this case; Playing the notification sound, vibrating and displaying a notification icon on the lock screen and in status bar.
**Expected result:** No sound and no vibration, only a notification icon on lock screen and status bar.

### Screenshots
![screenshot_2016-04-13-12-56-37_cropped](https://cloud.githubusercontent.com/assets/7093365/14491406/d8a6bd66-0178-11e6-9c29-984953058c28.png)

### Device info

**Device:** Fairphone FP2 (FP2)
**Android version:** 5.1 (r4275.1_FP2_gms43_1.2.8, r4275.1_FP2_gms43_1.2.8)
**Signal version:**  3.15.2

### Link to debug log
https://gist.github.com/anonymous/79ce42ea2581410f71b09424a34f8f85

**Comment by 2-4601** [comment]
Unable to reproduce on Nexus 4, stock 5.1.1, Signal 3.15.2

**Comment by mantono** [comment]
It turns out that this was misconfiguration in the Android settings that are (I think) specific to Fairphone 2.

I had selected priority interruption for Signal, because I thought that if I did not do that my starred contacts would not trigger a notification at all in priority mode. This was not the case. Signal should be set to normal mode, and selecting contacts as starred/favourite would override that setting for those contacts.

**Incorrect setting**
![signal1](https://cloud.githubusercontent.com/assets/7093365/14589335/c4683810-04df-11e6-89f5-9919ba3fa268.png)

**Correct setting**
![signal2](https://cloud.githubusercontent.com/assets/7093365/14589343/db8bb15c-04df-11e6-9425-1d9c6ddef6de.png)

So no bug in Signal after all :)
